astro/src/content/blog/authcon-press-release.mdx

```diff
@@ -0,0 +1,51 @@
+
+Maintaining in-house authentication systems is a significant security challenge. A [recent report](https://www.verizon.com/business/en-gb/resources/reports/dbir/?trk=article-ssr-frontend-pulse_little-text-block) shows that 35% of breaches involve stolen credentials, with web application credentials being three times more likely to be exploited than phishing or vulnerabilities. FusionAuth created [AuthCon](https://authcon.io) to help teams navigate these security risks and focus more of their efforts on value-driving activities.
+
+Topher Marie, Founder of Strata and JumpCloud, says, “In today’s engineering landscape, customer identity can no longer be a side topic. It’s at the heart of how organizations deliver seamless, secure, and scalable experiences to their users. Bringing engineering and product teams together to focus specifically on customer identity—separate from broader IAM or security conversations—creates a unique space to tackle the real-world challenges and opportunities in authentication and identity management.”
+
+## Adapting CIAM Strategies to New Engineering Realities
+
+The relationship between CIAM and engineering has evolved over the last decade. Technologies like Kubernetes and containers have shifted organizational dynamics, positioning application engineers as key revenue drivers. As a result, every authentication decision now impacts business success. [AuthCon](https://authcon.io) will focus on how teams can integrate CIAM with these emerging technologies and changing dynamics.
+
+Confirmed speakers at [AuthCon](https://authcon.io): 
+- Michael Gruen, VP of Engineering at Layr 
+- Matthew Goodrich, Director of Information Security at Alteryx 
+- Mike Ross, Director of Information Security at Mythical Games 
+- Chris Obdam, Sr., Director of Technologies at Betty Blocks 
+- Hannah Suor, Principal Product Manager at GitLab
+- Heather Flanagan, Principal at Spherical Cow Consulting and Executive Director at IDPro 
+- Cheryl Hung, Sr. Director of Infrastructure Ecosystems at Arm
+- Topher Marie, Founder at Strata & JumpCloud
+- Hirsch Singhal, Staff Product Manager at GitHub.
+
+[Register for AuthCon 2025](https://ticket.apidays.global/event/apidays-nyc-2025/207d2df0-83d4-4511-a6ae-3e1229e584b8/AuthCon2025) today to secure your spot, and don’t miss the opportunity to shape the conversation by submitting a proposal for our [Call for Presentations (CFP)](https://apidays.typeform.com/to/jyEoGGg1?utm_source=xxxxx&utm_medium=xxxxx&utm_campaign=xxxxx&typeform-source=www.authcon.io#event_name=xxxxx). Join leading industry experts, developers, and engineers as we explore the future of CIAM and authentication.
```

Comment:
Did you want the `xxxx` values for the utm params?

Reply:
This url is from apidays, I don't know about the xxxx values, but this is the url that seems to work as a hyperlink to the registration form we've made with them